workflow/gs-stateful-processes.adoc

```diff
@@ -193,40 +341,23 @@ A set of DDL (Data Definition Language) scripts for both supported databases (Po
 * Runtime Engine
 * User Tasks
 
-The script creates all the tables required by the features listed above to work. 
-
-The scripts are provided for the PostgreSQL database only.
-// what about MS SQL server?
-
-You can use standard PostgreSQL functionality to apply the scripts, such as:
-
-* pgAdmin Query Tool
-* A psql command (e.g., psql -h host -U username -d myDataBase -a -f myInsertFile)
+The script creates all the tables required by the features listed above to work.
 
 The scripts can be executed in any order, and the required tables will be created in the database that will be used by the {BAMOE} process service. 
 
-As well as manually executing the scripts you can use Flyway to automate the process of creating and maintaining database tables across patch releases. To avoid manual managament of the above scripts and their future update. To enable this feature, the following properties should be added in your project `application.property` file, which is found in `[your-project]/src/main/resources/application.properties`:
+In Development environments, you can use Kie Flyway to automate the process of creating and maintaining database tables across patch releases.
+To avoid manual management of the above scripts and their future update. To enable this feature, the following property
+should be added in your project `application.property` file, which is found in `application.properties`:
 
```

Comment:
maybe:
...should be added in your project's `application.properties` file, which is found in `src/main/resources`:

Reply:
good catch, it was there but I rewrote this :)